contrib/terraform/vsphere/output.tf

```diff
@@ -0,0 +1,31 @@
+output "master_ip_addresses" {
+  value = "${module.kubernetes.master_ip}"
```

Comment:
I get:

```
$ terraform0.13.5 validate

Warning: Interpolation-only expressions are deprecated

  on output.tf line 2, in output "master_ip_addresses":
   2:   value = "${module.kubernetes.master_ip}"
```

Can you please fix all `terraform validate` errors with TF 0.13 and TF0.14?

Also, please add a CI/CD pipeline like [here](https://github.com/kubernetes-sigs/kubespray/commit/b77460ec3480868530b6c0b9435d6e900edd8fe4#diff-dc1b2c31bdb8c19a596a37a04254d296b4d7b3cc96d75ac4bd7bae3474d1d474).

Reply:
Validated now with `v0.13.5` and  `v0.14.7`